### PR TITLE
Add config setting to disable file attachments

### PR DESCRIPTION
**Pull request by jasonblais**
https://github.com/mattermost/platform/pull/6301/

### CODE REVIEW

**Comment by jasonblais** [comment]
^this is for a PR that was merged right before code complete and didn't have `Docs: Needed` label added

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1824,6 +1824,14 @@ Secure Amazon S3 Connections
 | This feature's ``config.json`` setting is ``"AmazonS3SSL": true`` with options ``true`` and ``false`` for above settings respectively.                               |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable File Attachments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When false, disables file and image uploads on messages.
```
Comment (JeffSchering) [inline source/administration/config-settings.rst]: Maybe this is better: "When false, users cannot upload and share files and images by attaching them to messages."

Comment (jasonblais) [inline source/administration/config-settings.rst]: Could there be too many `and`'s for a non-native English speaker?

Comment (JeffSchering) [inline source/administration/config-settings.rst]: Maybe. 0/5 on whether or not you make the changes.

Comment (jasonblais) [inline source/administration/config-settings.rst]: updated, let me know what you think

**Pull request merged**